packages/gestalt/src/SideNavigation/GroupContent.js

```diff
@@ -12,7 +12,7 @@ import Text from '../Text';
 import { type Indexable } from '../zIndex';
 
 type IconType = $Keys<typeof icons> | { __path: string };
-type Display = 'expandable' | 'static';
+type Display = 'expandable' | 'expanded' | 'static';
```

Comment:
Wouldn't it be cleaner if expanded was it's own prop? 

Like `<Sidenavigation.Group type="expandable"  expanded />`

Expandable vs Expanded seems a little confusing, and the states don't seem that different no?

Reply:
Expanded seems like a forced override essentially

Reply:
I agree w ```<Sidenavigation.Group type="expandable"  expanded />```

My only caveat is ```<Sidenavigation.Group type="static"  expanded />``` this now becomes possible.

Reply:
That's ok right? It's expanded anyways. It's a dead prop at that point

Reply:
Since all static displays are expanded by default

Reply:
i think i prefer 

`'static' | 'expandable' | 'expandableExpanded'`


1. We avoid booleans
2. We avoid dead code
3. expandableExpanded is more explicit

I thought about expandable > expandableCollapsed but... that's a breaking change. but we could discuss.

Reply:
That seems confusing, are you sure?

It seems to me, just from looking at the examples, that expandable means -> collapse icons are showing vs they are not showing. 

expandableExpanded seems like a lot,

`Expanded` is a state variable. Essentially, we're making it a controlled component. It's like `checked`.  In static mode, there are no "expand" states, so it's not that important. Maybe there's a flow type to disable that case

Reply:
we are not making it a controlled component, we are just setting a new initial state that controls if a group should be be initialized (collapsed vs expanded). But the user can still go and change that and it's still uncontrolled. 

The engineer shouldn't keep the state collapsed/expanded outside, it's uncontrolled to that respect.

Reply:
Actually I see... some other improvement here.

Reply:
I see your point now. With expanded prop we can control them. I thought just have them open from the beginning, but posterior changes don't persist. so there's no real differences between that and static.

Reply:
it probably makes more sense to control it. There are many implementations to control that. Then we need on onExpand too there.